pkg/api/serialization_test.go

```diff
@@ -125,7 +125,7 @@ func TestList(t *testing.T) {
 	roundTripSame(t, item)
 }
 
-var nonRoundTrippableTypes = util.NewStringSet("ContainerManifest", "ContainerManifestList")
+var nonRoundTrippableTypes = util.NewStringSet()
 var nonInternalRoundTrippableTypes = util.NewStringSet("List", "ListOptions", "PodExecOptions")
```

Comment:
@nikhiljindal @bgrant0607 , can we remove `nonRoundTrippableTypes`? Or are we expecting there will be new member in this sets in the future?